turbo/apps/web/app/api/agent/runs/resume/route.ts

```diff
@@ -1,188 +1,42 @@
-import { NextRequest } from "next/server";
-import { initServices } from "../../../../../src/lib/init-services";
-import { agentRuns } from "../../../../../src/db/schema/agent-run";
-import { agentConfigs } from "../../../../../src/db/schema/agent-config";
-import { runService } from "../../../../../src/lib/run";
-import { getUserId } from "../../../../../src/lib/auth/get-user-id";
-import { generateSandboxToken } from "../../../../../src/lib/auth/sandbox-token";
-import {
-  successResponse,
-  errorResponse,
-} from "../../../../../src/lib/api-response";
-import {
-  BadRequestError,
-  UnauthorizedError,
-} from "../../../../../src/lib/errors";
-import { eq } from "drizzle-orm";
-import {
-  sendVm0StartEvent,
-  sendVm0ErrorEvent,
-} from "../../../../../src/lib/events";
-
-export interface ResumeAgentRunRequest {
-  checkpointId: string;
-  prompt: string;
-}
-
-export interface ResumeAgentRunResponse {
-  runId: string;
-  status: string;
-  createdAt: string;
-}
+import { NextRequest, NextResponse } from "next/server";
 
 /**
  * POST /api/agent/runs/resume
- * Resume execution from a checkpoint
+ *
+ * @deprecated Use POST /api/agent/runs with { checkpointId, prompt } instead.
+ * This endpoint redirects to the unified runs API.
  */
 export async function POST(request: NextRequest) {
```

Comment:
I think there's no need to keep "redirect"; we can just delete it. Let's not do backward compatibility for now.